README.md

```diff
@@ -105,15 +105,31 @@ always in sync, and that one or the other may be slightly more up-to or out-of d
 
 ### `review.tsv`
 Columns:
-- `classCode`: integer: ID of review case class
-- `classShortName`: string (camelCase): describing the review case class
+- `classCode`: integer
+- `classLabel`: string
 - `value`: any: Some form of data to review
 - `comment`: string (optional)
 
-#### 1. `causalD2gButMarkedDigenic`
-This review case involves what would be otherwise considered a valid disease-gene relationship, but for the fact that 
-it quite unusually includes 'digenic' in the label, even though it only had 1 association. OMIM doesn't have a 
+#### 1. D2G Disease-defining but marked digenic
+This review case involves what would be otherwise considered a valid disease-gene (D2G) relationship, but for the fact 
```

Comment:
There's a lot of commentary scattered throughout the codebase, but not really anything in the README / front page that explains this stuff.  So it will entail a new section; I'm not sure what to call it. Maybe an "additional information" or "topics" or "explanation" section with subsection "Disease-Gene associations".

Reply:
A new section in the README is fine

Reply:
@twhetzel I updated the README. I'm not sure if I'm done yet; check it out.

Reply:
I'd prefer to review when you have added everything. Is there some specific feedback you are looking for about the README? The additions I requested should only take ~20 min to add.

Reply:
I'll finish it up and then tag you again. I took is taking a little more time than that. You've been wanting a lot of related documentation for a while, so I compiled it all in a way I felt you and the team would like, phrased in a way that is as clear / least confusing as I can make it (harder to do in this case than other things I typically document).

Reply:
Just re-checked and these `README.md` updates are done.
- [x] Addition of new section: _Under the hood: Design decisions, etc._
  - Has only one sub-section: Gene-Disease Pipeline
    - Include's Trish's request: _can you add a section that includes what a "valid disease-gene (D2G) relationship"_
    - Additional content to support that disease-defining explanation, and non-disease defining stuff (copy/pasted from code notes and reformatted)
      - Example input/output
      - Inverse predicate for disease-defining (`RO:0004003` and `RO:0004013`)
      - Other non-disease-defining gene-disease association predicates.

Reply:
@joeflack4 did you intend the README file to be in this PR? You mentioned above there are updates to it.